Find a functionally equivalent code:com.android.keyguard.KeyguardViewBase.show:COMMENT
<com.android.keyguard.KeyguardViewBase: void show()>
Method Modifier: public      abstract    
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Called when the view needs to be shown.
 */

@@ -1 +1,5 @@
-
+{
+    if (DEBUG)
+        Log.d(TAG, "show()");
+    mSecurityContainer.showPrimarySecurityScreen(false);
+}

